### ISSUE TITLE
Operators missing for MXNet backend

### ISSUE THREAD

**Issue by sandeep-krishnamurthy** [deep-learning-tools/keras#18]
# Update Operators
- [ ] update
- [ ] update_add
- [ ] update_sub

# Graph Manipulations
- [ ] gradients

# RNNs
- [ ] rnn

# CNNs
- [ ] conv1d
- [ ] conv2d_transpose
- [ ] separable_conv2d
- [ ] depthwise_conv2d
- [ ] conv3d
- [ ] conv3d_transpose
- [ ] local_conv1d
- [ ] local_conv2d

# Higher Order Functions
- [ ] map_fn
- [ ] foldl
- [ ] foldr

# Sparse Tensors
Sparse Tensors are not supported in any operators.

# NN Operators
- [ ] sparse_categorical_crossentropy

# Others
- [ ] cumsum
- [ ] cumprod
- [ ] logsumexp
- [ ] stack

**Comment by Robchio** [comment]
Sorry for the obvious question - what's the meaning of a 'checked box' vs. an empty box? 

Has 'checked' been implemented? For example, does keras-apache-mxnet support conv1d model layers or not?

**Comment by kalyc** [comment]
`checked` means implemented. So all of above operators containing a `check` mark beside their name are supported

**Comment by Cpruce** [comment]
Hi, I need the cumsum symbol for a loss function I'm adding. Is this currently being worked on? I'm starting to add it to src/operator/tensor now
